app/jobs/dqt/sync_teachers_job.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module Dqt
+  class SyncTeachersJob < ApplicationJob
+    queue_as :dqt_sync
+
+    def perform(batch_size = 500, interval = 30.seconds)
```

Comment:
I thought 500 was a safe number so as to not affect the quota too much but I'm happy to lower it or increase it.